inc/widgets/odm-custom-posts-widget.php

```diff
@@ -93,7 +91,9 @@ public function widget( $args, $instance ) {
 						$index++;
 					endforeach;
 					?>
-
+				<div class="sixteen columns align-right">
```

Comment:
@S-mardii here, you are adding ```<div class="sixteen columns align-right">``` inside another ```<div class="sixteen columns">```. I would suggest you to not nest them but put them at the same level

```
<div class="sixteen columns">
</div>
 <div class="sixteen columns align-right"> 
</div>
```